packages/theme/src/theme.ts

```diff
@@ -68,11 +68,10 @@ export enum ThemeVars {
   FontSize3xl = '--s-font-size-3xl',
   FontSize4xl = '--s-font-size-4xl',
 
-  // // ???
-  // SizeMini = '--s-size-mini',
-  // SizeSmall = '--s-size-small',
-  // SizeMedium = '--s-size-medium',
-  // SizeBig = '--s-size-big',
+  SizeMini = '--s-size-mini',
+  SizeSmall = '--s-size-small',
+  SizeMedium = '--s-size-medium',
+  SizeBig = '--s-size-big',
```

Comment:
I am really not sure that these variables should exist in theme. It's not determined about which size exactly they are. I am looking at Figma and thinking that such variables should be better:

Radio, Checkbox:

- `12px` - `SizeMd`
- `16px` - `SizeLg`
- `20px` - `SizeXl`

Select, Input, Button:

- `24px` - `ControlHeightSm`
- `32px` - `ControlHeightMd`
- `40px` - `ControlHeightLg`
- `56px` - `ControlHeightXl`

Also please use "sm-md-xl" notation instead of "mini-small-medium etc"

Anyway, even these vars aren't clear, imo.